Azure/azure-cli
# [RDBMS] `az mysql flexible-server update`: Expose `--geo-redundant-backup` argument

**Related command**
`az mysql flexible-server update`

**Description**
This PR enables the argument `--geo-redundant-backup` when updating a MySQL flexible server with possible values `Enabled` and `Disabled`.

**Testing Guide**
- Create MySQL flexible server without geo redundant backup: `az mysql flexible-server create -g resourceGroup -n serverName -l location --public-access none --tier GeneralPurpose --sku-name Standard_D2ds_v4`
- Enable geo redundant backup: `az mysql flexible-server update -g resourceGroup -n serverName --geo-redundant-backup Enabled`
- Disable geo redundant backup: `az mysql flexible-server update -g resourceGroup -n serverName --geo-redundant-backup Disabled`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
